artifacts/rhdh-plugins-reference/argocd/argocd-plugin-readme.adoc

```diff
@@ -1,322 +1,191 @@
-
-[source,console]
-----
-kubectl patch deployment argocd-server --type "json" -p '[{"op":"add","path":"/spec/template/spec/containers/0/command/-","value":"--insecure"}]'
-----
-
-Also, you can use Helm to install the ArgoCD plugin and be insecure by default:
-
-[source,console]
-----
-helm upgrade --install argocd argo/argo-cd \
-  --version 3.33.5 \
-  --set 'server.extraArgs={--insecure}'
-----
-
-== For users
-
-=== Using ArgoCD plugin in {product}
+[NOTE]
+====
+If you do not set these annotations, the plugin defaults to the first ArgoCD instance configured in `app.config.yaml`.
+====
 
-The ArgoCD plugin displays the current state of an application in Backstage Catalog.
+== For Users
 
 .Prerequisites
 
-* Your Backstage application is installed and running.
-* You have installed the ArgoCD plugin. For installation steps, see <<Installation>>.
+* Use the following configuration to install the ArgoCD plugin into Red Hat Developer Hub using Helm:
 
-.Procedure
-
-. Open your Backstage application and go to the *Catalog* page.
-. In the *OVERVIEW* tab, you can see the ArgoCD overview card at the bottom.
 +
---
-
-image::argo-cd-plugin-overview-card.png[ArgoCD overview card]
+[source,yaml]
+----
+global:
+  dynamic:
+    includes:
+      - dynamic-plugins.default.yaml
+    plugins:
+      - package: ./dynamic-plugins/dist/roadiehq-backstage-plugin-argo-cd-backend-dynamic
+        disabled: false
+----
 
-The ArgoCD overview card displays application information including *NAME*, *INSTANCE*, *SYNC STATUS*, *HEALTH STATUS*, and *LAST SYNCED*.
---
+.Procedures
 
-. To view the detailed application information, select an application from the ArgoCD overview card.
-+
---
-A pop-up containing detailed information about the application appears, along with a link to ArgoCD dashboard.
+. On the RHDH platform, navigate to the *Catalog* and select an appropriate component that you have moved to production.
 
-image::argo-cd-plugin-overview-card-details.png[ArgoCD overview card]
---
+. Select *CD* tab to view insights into deployments managed by ArgoCD plugin. 
 
-. To view the ArgoCD history, go to the *CD* tab in the Catalog.
 +
---
-
-image::argo-cd-plugin-history.png[ArgoCD overview card]
-
-The *ArgoCD history* page contains history of application instances along with other information, such as deploy details, author name, message that author added, and revision ID.
---
-
-
+image::rhdh-plugins-reference/argocd.png[CD tab ArgoCD]
 
+[NOTE]
+====
+For additional deployment details, select an appropriate card or select the link icon (image:rhdh-plugins-reference/link.svg[width=12px, height=12px, align=center]) to open the deployment details in *argo*.
+==== 
 
+. (Optional) Alternatively, select *Overview* tab, and then from Deployment summary section, select an appropriate application to open the deployment details in *argo*.
```

Comment:
```suggestion
 *Overview* tab contains the Deployment summary, select an appropriate application to open it in *Argo CD*.
```

Can we also add the screenshot of the Overview tab here ?


![image](https://github.com/user-attachments/assets/d3563423-dc01-4fe4-8297-1664a1d69357)